src/main/webapp/WEB-INF/category.jsp

```diff
@@ -70,6 +83,7 @@
     <%    }
         }  %>
     </div>
-
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBCK_yt5P_kfz23tAb8tE_fptjRAn5jaB0&callback=initMap">
```

Comment:
I think you can remove the "&callback=initMap" since you don't appear to have an initMap JavaScript function to call.

Reply:
You're right, done